win32ss/user/user32/windows/class.c

```diff
@@ -1466,32 +1466,29 @@ RegisterClassExWOWW(WNDCLASSEXW *lpwcx,
     }
 */
     RtlInitEmptyAnsiString(&AnsiMenuName, NULL, 0);
-    if (WndClass.lpszMenuName != NULL)
+    if (WndClass.lpszMenuName && !IS_INTRESOURCE(WndClass.lpszMenuName))
```

Comment:
This doesn't seem to change anything. If `WndClass.lpszMenuName` is `NULL`, `MenuName` and `AnsiMenuName` now get assigned a `NULL` buffer, which they were already initialized with.